tests/sentry/lang/javascript/example-project/launch.js

```diff
@@ -4,8 +4,8 @@ const stacktraceParser = require('stacktrace-parser');
 
 function convertStack(frames) {
   return frames.map((item) => {
-    var match = item.file.match(/^.*\/(.*?)$/);
-    var fileName, absPath;
+    const match = item.file.match(/^.*\/(.*?)$/);
```

Comment:
Honestly, I'm not familiar with where this file is actually used, I'd be more comfortable if you reverted changes to this file.

Reply:
In this file, `var`, `const` and `let` are mixed. So I changed `var`s to `const` or `let` to be better. Hmm.. 
but okay, I will revert this file.